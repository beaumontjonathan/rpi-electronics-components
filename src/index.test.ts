function add(x: number, y: number) {
  return x + y;
}

describe('index', () => {
  it('should pass', () => {
    // expect(add(1, 2)).toBe(4);
  });
});
