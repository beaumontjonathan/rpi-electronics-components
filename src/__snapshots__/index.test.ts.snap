// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should match snapshot 1`] = `
Object {
  "BUTTON_STATE": Object {
    "DOWN": 1,
    "UNKNOWN": -1,
    "UP": 0,
  },
  "Button": [Function],
}
`;
